### ISSUE TITLE
Checkpoint not found - Examples

### ISSUE THREAD

**Issue by naman5jain4** [yu45020/Text_Segmentation_Image_Inpainting#8]
I was attempting to run the demo code, but I found that the checkpoint file mentioned in line 53 from "Examples/demo_segmentation.py" (see below) is unavailable. Is there a location that has not been documented that I can check? 
`old = torch.load("checkpoints/text_seg_model_838epos.pt", map_location='cpu')`

**Comment by yu45020** [comment]
You may download checkpoints [here](https://github.com/yu45020/Text_Segmentation_Image_Inpainting/tree/master/checkpoints).  Uploading them to github seems less practical.

**Comment by naman5jain4** [comment]
I completely agree with the practicality, but the Google Drive folder contains a model created at 681 epochs and not 838 epochs. Could you please verify the model has been uploaded?
![capture](https://user-images.githubusercontent.com/21376771/53450878-6b314680-39eb-11e9-9fbb-3f19c1ae4e0c.PNG)

**Comment by yu45020** [comment]
Sorry for the confusion. I am not able to find the newer checkpoint Orz. 

I am currently testing variance configuration of [Atrous Convolutional Neural Network (ACNN) for Semantic Image Segmentation with full-scale Feature Maps](https://arxiv.org/abs/1901.09203). It shows far better than Xecption based model.

**Comment by BarryStark** [comment]
I was attempt to run the demo code , but the address you provided(https://drive.google.com/open?id=1iiWjf-PKBq_nfD9lqKtuyU4bqEbUaq_5) didn't work, where can I find a suitable model?I would appreciate it if you can reply as soon as possible.

**Comment by BarryStark** [comment]
突然发现是国人😂

**Comment by yu45020** [comment]
重新練一個吧，這項目目前擱置了。

**Comment by BarryStark** [comment]
您的邮件已收到，感谢来信！

**Comment by IamMegatron2025** [comment]
> https://drive.google.com/open?id=1iiWjf-PKBq_nfD9lqKtuyU4bqEbUaq_5

@yu45020  您好，这个模型pt的链接失效了 能否再更新下连接？

**Comment by BarryStark** [comment]
您的邮件已收到，感谢来信！

**Comment by yu45020** [comment]
> > https://drive.google.com/open?id=1iiWjf-PKBq_nfD9lqKtuyU4bqEbUaq_5
> 
> @yu45020 您好，这个模型pt的链接失效了 能否再更新下连接？

@PapaMadeleine2022 你好，这项目鸽了好久了，pt我也没有留底😂。 推荐换个模型重新练。

**Comment by arthurwolf** [comment]
> You may download checkpoints [here](https://github.com/yu45020/Text_Segmentation_Image_Inpainting/tree/master/checkpoints). Uploading them to github seems less practical.

The link is now broken...

**Comment by BarryStark** [comment]
您的邮件已收到，感谢来信！
